iOSNativeLoginTemplate/Podfile

```diff
@@ -7,11 +7,13 @@ target 'iOSNativeLoginTemplate' do
   source 'https://cdn.cocoapods.org/'
   use_frameworks!
   use_mobile_sdk!
+  
+  pod "RecaptchaEnterprise", "18.5.0-beta01"
```

Comment:
This is the version shown in Google's documentation and I'm pretty used to "beta" meaning ready for production in Google-speak.  We could back up to an older stable if anyone see merit.

Reply:
Yeah, to them alpha is usually "production quality but we might make API changes".  Beta is pretty much always safe.